graphql_server/__init__.py

```diff
@@ -1,6 +1,7 @@
 from copy import deepcopy
 from datetime import date, datetime
-from threading import Lock
+from tokenize import Single
```

Comment:
What does this do and where is it used? I don't see any reference to `Single` in the code.

Reply:
VS Code add some dumb stuff to autocomplete. Thanks, I'll delete it

Reply:
LOL happens to all of us sometimes.